src/index.js

```diff
@@ -395,6 +405,14 @@ class ReactNativeModal extends Component {
       : this.props.swipeDirection === direction;
   };
 
+  getThreshold = direction => {
+    if (typeof this.props.swipeThreshold === 'number') {
+      return this.props.swipeThreshold;
+    }
+
+    return this.props.swipeThreshold[direction];
```

Comment:
This will return `undefined` if the direction isn't defined into `swipeTreshold`. We should issue a warning and return 0.